apps/web/app/(app)/environments/[environmentId]/settings/billing/actions.ts

```diff
@@ -24,6 +25,8 @@ export async function upgradePlanAction(
   if (!isAuthorized) throw new AuthorizationError("Not authorized");
 
   const subscriptionSession = await createSubscription(teamId, environmentId, priceLookupKeys);
+
+  teamCache.revalidate({ id: teamId });
```

Comment:
why do we need to revalidate the cache here? We should only revalidate the cache after making the database call that changes the team object. Where is the call made? Can we move the `revalidate` command after the db call instead?

Reply:
ahh apologies! it already happens inside the `updateTeam` service called inside the ee/billing method! Same for below, removed the both. Pushed 🤞🏼